unittests/finalizer_tests.cpp

```diff
@@ -154,8 +168,308 @@ BOOST_AUTO_TEST_CASE( finalizer_safety_file_io ) try {
 
 } FC_LOG_AND_RETHROW()
 
```

Comment:
Really seems like we are starting a whole diff section of tests. Maybe move to `finalizer_votes_tests.cpp`.

Reply:
Will do!

Reply:
done